docs/source/appendices/firewall-notes.md

```diff
@@ -0,0 +1,60 @@
+# Notes on Firewall Setup
+
+This page summarizes the ports used by BigchainDB and the traffic they should accept or reject, e.g. using a firewall or AWS security group.
+
+
+## Inbound at Port 22
+
+Port 22 is the default SSH port (TCP). It should expect unsolicited inbound traffic from arbitrary IP addresses.
+
+
+## Inbound at Port 123
```

Comment:
We don't need a rule to allow `NEW` packets going through `iptables`. I'd move this section to **Outbound Traffic**, saying that if your firewall policy is to explicitly drop packets going through the `OUTPUT` chain, then you must remember to enable the port `123`.